### PR TITLE
Add updated Facebook Pay icon

### PR DESCRIPTION
**Pull request by csgavino**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [X] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [X] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by csgavino** [comment]
Web:
Screen Shot 2021-08-05 at 10 53 08
Mobile:
Screen Shot 2021-08-05 at 10 53 25

**Comment by csgavino** [comment]
Screen Shot 2021-08-05 at 10 56 21

**Comment by larouxn** [comment]
Looks like the new icon has an outer border or something that's causing it to appear larger than other icons. I _think_ it's an inner/outer border setting issue.

old FB, new FB, Visa
Screen Shot 2021-08-05 at 11 18 46
Screen Shot 2021-08-05 at 11 18 58

File: app/assets/images/payment_icons/facebook_pay.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...-2.023v3.014h2.022c.648 0 1.12-.125 1.418-.377.297-.25.445-.627.445-1.13 0-.502-.148-.879-.445-1.13-.298-.251-.77-.377-1.418-.377z" fill="#1B2529"/><defs><linearGradient id="pi-paint0_linear" x1="8.325" y1="6.803" x2="3.828" y2="10.121" gradientUnits="userSpaceOnUse"><stop stop-color="#24D365"/><stop offset="1" stop-color="#139DBD"/></linearGradient><linearGradient id="pi-paint1_linear" x1="8.29" y1="6.774" x2="12.8" y2="10.111" gradientUnits="userSpaceOnUse"><stop stop-color="#23D068"/><stop offset=".52" stop-color="#EACD36"/><stop offset="1" stop-color="#FF6C5C"/></linearGradient><linearGradient id="pi-paint2_linear" x1="5.121" y1="15.876" x2="11.496" y2="15.876" gradientUnits="userSpaceOnUse"><stop stop-color="#266FF3"/><stop offset="1" stop-color="#C439D6"/></linearGradient><linearGradient id="pi-paint3_linear" x1="4.135" y1="15.894" x2="2.281" y2="10.39" gradientUnits="userSpaceOnUse"><stop stop-color="#266FF3"/><stop offset="1" stop-color="#149FBA"/></linearGradient><linearGradient id="pi-paint4_linear" x1="12.899" y1="9.913" x2="11.041" y2="15.394" gradientUnits="userSpaceOnUse"><stop stop-color="#FF6C5C"/><stop offset="1" stop-color="#C439D6"/></linearGradient></defs></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/facebook_pay.svg:1]: We need to:

- Remove the `fill` attribute on the SVG level
- Remove the `<path xmlns="http://www.w3.org/2000/svg" d="M36.304 0H1.696C.76 0 0 .746 0 1.667v20.666C0 23.253.76 24 1.696 24h34.608C37.24 24 38 23.254 38 22.333V1.667C38 .747 37.24 0 36.304 0z" fill="#fff" stroke="#000" stroke-opacity=".07"/>` element
- Add the white background `<path xmlns="http://www.w3.org/2000/svg" fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/>`

```suggestion
<svg width="38" height="24" role="img" aria-labelledby="pi-facebook_pay" viewBox="0 0 38 24" xmlns="http://www.w3.org/2000/svg"><title id="pi-facebook_pay">Facebook Pay</title><path xmlns="http://www.w3.org/2000/svg" opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path xmlns="http://www.w3.org/2000/svg" fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><path fill-rule="evenodd" clip-rule="evenodd" d="M8.29 5.672h.035v.874H8.29c-2.256 0-4.165 1.46-4.8 3.472l-.849-.265c.75-2.365 2.995-4.081 5.649-4.081z" fill="url(#pi-paint0_linear)"/><path fill-rule="evenodd" clip-rule="evenodd" d="M8.325 6.546v-.874H8.29c2.662 0 4.914 1.728 5.656 4.105l-.848.266c-.624-2.014-2.523-3.482-4.773-3.497z" fill="url(#pi-paint1_linear)"/><path fill-rule="evenodd" clip-rule="evenodd" d="M11.243 15.478l.523.708a5.96 5.96 0 01-3.476 1.108 5.96 5.96 0 01-3.43-1.075l.524-.708c.82.573 1.823.91 2.906.91 1.104 0 2.124-.35 2.953-.943z" fill="url(#pi-paint2_linear)"/><path fill-rule="evenodd" clip-rule="evenodd" d="M2.642 9.753l.848.265a4.856 4.856 0 00-.225 1.465c0 1.663.836 3.134 2.119 4.028l-.523.708c-1.453-1.017-2.417-2.663-2.483-4.532v-.407c.019-.53.11-1.042.264-1.527z" fill="url(#pi-paint3_linear)"/><path fill-rule="evenodd" clip-rule="evenodd" d="M14.205 11.483c0 1.934-.96 3.646-2.44 4.703l-.523-.707a4.904 4.904 0 002.073-3.996c0-.502-.076-.987-.218-1.443l.848-.266c.17.54.26 1.114.26 1.71z" fill="url(#pi-paint4_linear)"/><path fill-rule="evenodd" clip-rule="evenodd" d="M11.06 12.218c0-.308-.215-.446-.692-.446H9.193v.89h1.175c.485 0 .692-.133.692-.444zm-.857-1.866h-1.01v.89h1.01c.457 0 .664-.138.664-.443 0-.306-.21-.447-.664-.447zm1.474 1.922c0 .594-.474.932-1.31.932H8.64c-.03 0-.05-.02-.05-.049v-3.3c0-.029.02-.048.05-.048h1.575c.813 0 1.26.313 1.26.878 0 .335-.15.604-.583.733.548.114.77.437.785.854zM7.872 9.857v.461c0 .03-.02.049-.05.049h-2v.861h1.66c.03 0 .049.02.049.049v.46c0 .03-.02.05-.05.05H5.823v1.37c0 .03-.02.049-.05.049H5.27c-.03 0-.05-.02-.05-.049v-3.3c0-.029.02-.048.05-.048h2.553c.03 0 .05.02.05.048z" fill="#64717D"/><path fill-rule="evenodd" clip-rule="evenodd" d="M31.214 9.82a49.278 49.278 0 001.688 4.454l1.55-4.454h1.239l-2.221 5.754c-.284.737-.621 1.305-1.011 1.705-.39.4-.913.6-1.568.6-.163 0-.317-.013-.46-.039v-1.005c.12.019.227.028.317.028.398 0 .722-.11.972-.332.25-.221.47-.546.659-.974a45.905 45.905 0 01-1.312-2.824 80.59 80.59 0 01-1.17-2.913h1.317zm-4.6-.134c.696 0 1.256.197 1.68.591.425.395.637.985.637 1.77V15.4h-1.09v-.91c-.22.328-.503.58-.847.757a2.542 2.542 0 01-1.176.265c-.599 0-1.064-.152-1.395-.455-.331-.303-.497-.703-.497-1.197 0-.491.187-.903.56-1.234.373-.33.968-.497 1.786-.497h1.545v-.016c0-.938-.439-1.407-1.317-1.407-.368 0-.677.065-.93.196-.251.13-.455.312-.61.547l-.778-.62c.242-.368.561-.651.957-.848a2.985 2.985 0 011.35-.296h.124zM20.16 7.588c.996 0 1.74.233 2.232.7.492.467.739 1.1.739 1.9 0 .797-.247 1.43-.739 1.898-.492.469-1.236.703-2.232.703h-2.136v2.612H16.83V7.588h3.329zm7.657 5.419h-1.5c-.446 0-.765.07-.957.21a.665.665 0 00-.286.566c0 .49.33.736.994.736.439 0 .804-.097 1.096-.293.292-.195.51-.46.653-.795v-.424zm-7.77-4.326h-2.023v3.014h2.022c.648 0 1.12-.125 1.418-.377.297-.25.445-.627.445-1.13 0-.502-.148-.879-.445-1.13-.298-.251-.77-.377-1.418-.377z" fill="#1B2529"/><defs><linearGradient id="pi-paint0_linear" x1="8.325" y1="6.803" x2="3.828" y2="10.121" gradientUnits="userSpaceOnUse"><stop stop-color="#24D365"/><stop offset="1" stop-color="#139DBD"/></linearGradient><linearGradient id="pi-paint1_linear" x1="8.29" y1="6.774" x2="12.8" y2="10.111" gradientUnits="userSpaceOnUse"><stop stop-color="#23D068"/><stop offset=".52" stop-color="#EACD36"/><stop offset="1" stop-color="#FF6C5C"/></linearGradient><linearGradient id="pi-paint2_linear" x1="5.121" y1="15.876" x2="11.496" y2="15.876" gradientUnits="userSpaceOnUse"><stop stop-color="#266FF3"/><stop offset="1" stop-color="#C439D6"/></linearGradient><linearGradient id="pi-paint3_linear" x1="4.135" y1="15.894" x2="2.281" y2="10.39" gradientUnits="userSpaceOnUse"><stop stop-color="#266FF3"/><stop offset="1" stop-color="#149FBA"/></linearGradient><linearGradient id="pi-paint4_linear" x1="12.899" y1="9.913" x2="11.041" y2="15.394" gradientUnits="userSpaceOnUse"><stop stop-color="#FF6C5C"/><stop offset="1" stop-color="#C439D6"/></linearGradient></defs></svg>
```

**Review by tauthomas01 — DISMISSED** [review]

**Comment by csgavino** [comment]
Screen Shot 2021-08-09 at 18 12 18
Screen Shot 2021-08-09 at 18 12 38

**Review by larouxn — APPROVED** [review]
LGTM! 🎩 
Screen Shot 2021-08-10 at 09 58 04

**Pull request merged**